mealie/schema/group/group_shopping_list.py

```diff
@@ -15,12 +15,7 @@
 
 class ShoppingListItemRecipeRef(MealieModel):
     recipe_id: UUID4
-    recipe_quantity: NoneFloat
-
-    # some recipes have a null quantity, so we default to 0
-    @validator("recipe_quantity")
-    def default_null_quantity(cls, v):
-        return v or 0
+    recipe_quantity: NoneFloat = 0
```

Comment:
I used the validator in case None wasn't an acceptable value in shopping list recipe refs, so it would convert None to 0. In case you're wondering why I did it this way haha. But if that doesn't matter this works too